addons/website_sale_collect_wishlist/__init__.py

```diff
@@ -0,0 +1 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
```

Comment:
:eyes:

Reply:
Got it! (note your link is not correct )

Reply:
Oops! My bad!